api/tm-catalog.openapi.yaml

```diff
@@ -53,6 +53,14 @@ paths:
           schema:
             type: string
           example: filter.mpn=B21,FC102
+        - name: 'filter.name'
+          in: query
+          description: |
+            Filters the inventory by TM name (consisting of 'author/manufacturer/mpn') having a prefix match.   
```

Comment:
In the open API we currently dont use the word TM.  
I think `...by inventory entry name` instead of `...by TM name` would be better.

Reply:
updated